### PR TITLE
Added macOS dock menu

### PR DESCRIPTION
**Pull request by israelroldan**
I have added a dock menu for macOS. This should also be useful for whenever #169 is implemented as it is standard practice to open new windows from the dock menu.

Preview:
screen shot 2017-04-10 at 12 17 11

### CODE REVIEW

File: main/main.js [hunk 1]
```diff
@@ -539,3 +540,36 @@ function createAppMenu () {
...
+      },
+      {
+        label: 'New Private Tab',
+        click: function (item, window) {
+          sendIPCToWindow(window, 'addPrivateTab')
+        }
+      },
+      {
+        label: 'New Task',
+        click: function (item, window) {
```
Comment (z3t0) [inline main/main.js]: What is a task.

**Comment by israelroldan** [comment]
Min calls groups of tabs "Tasks".

**Comment by z3t0** [comment]
@israelroldan Ah thanks that makes sense

**Comment by PalmerAL** [comment]
Thanks for doing this! I've merged this with a few minor style fixes in #344.

**Pull request closed without merging**